controllers/grants/index.js

```diff
@@ -269,20 +259,23 @@ router.get('/related', injectCopy('funding.pastGrants.search'), async (req, res,
 
 router.get('/recipients/:id', injectCopy('funding.pastGrants.search'), async (req, res, next) => {
     try {
-        let qs = addPaginationParameters({}, req.query.page);
-        qs.recipient = req.params.id;
-        qs.locale = res.locals.locale;
+        const query = {
+            page: req.query.page || 1,
+            recipient: req.params.id,
+            locale: res.locals.locale
+        };
+
         const data = await request({
             url: PAST_GRANTS_API_URI,
             json: true,
-            qs: qs
+            qs: query
         });
 
         const paginationLabels = req.i18n.__('global.misc.pagination');
 
-        if (data && data.meta.totalResults > 0) {
-            const firstResult = head(data.results);
-            const organisation = head(firstResult.recipientOrganization);
+        const organisation = get(data, 'results[0].recipientOrganization[0]');
```

Comment:
Using `get` for the whole thing (turns out you can pluck array items out using this) to avoid any red herrings from trying to get the organisation.